src/auth/src/credentials/service_account_credential.rs

```diff
@@ -66,11 +68,17 @@ impl TokenProvider for ServiceAccountTokenProvider {
     async fn get_token(&self) -> Result<Token> {
         let signer = self.signer(&self.service_account_info.private_key)?;
 
-        let claims = JwsClaimsBuilder::default()
-            .iss(self.service_account_info.client_email.as_str())
-            .scope(DEFAULT_SCOPES)
-            .build()
-            .map_err(CredentialError::non_retryable)?;
+        let now = OffsetDateTime::now_utc() - CLOCK_SKEW_FUDGE;
+        let exp = now + DEFAULT_TOKEN_TIMEOUT;
+        let claims = JwsClaims {
+            iss: self.service_account_info.client_email.clone(),
+            scope: Some(DEFAULT_SCOPES.map(|s| s.to_string()).to_vec()),
+            aud: None,
+            exp,
+            iat: now,
+            typ: None,
+            sub: None,
+        };
```

Comment:
Alternatively, you could create a `JwsClaims::new(now: OffsetDateTime) -> Self ` function.